lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -172,6 +175,43 @@ def add_files_to_build_phases
             end
           end
 
+          # Adds the test app host targets for the library to the Pods project with the
+          # appropriate build configurations.
+          #
+          # @return [void]
+          #
+          def add_test_app_host_targets
+            target.test_specs.each do |test_spec|
+              next unless test_spec.consumer(target.platform).requires_app_host?
+              name = target.app_host_label(test_spec.test_type)
+              platform_name = target.platform.name
+              app_host_target = !project.targets.find { |t| t.name == name }
```

Comment:
this is wrong. There should be no `!` there. Internal testing caught this. fixing.